devops/containers/ubuntu2204_build.Dockerfile

```diff
@@ -13,9 +13,7 @@ COPY scripts/install_build_tools.sh /install.sh
 RUN /install.sh
 
 RUN apt install -yqq libnuma-dev wget gnupg2 && \
-  apt-get install -yqq gcc-12 g++-12 libstdc++-12-dev && \
-  update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-12 100 && \
-  update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-12 100
+  apt-get install -yqq libstdc++-12-dev
```

Comment:
Can it be just `libstdc++-dev` to match the system gcc installation?

Reply:
I'll try it.

Reply:
Unfortunately it doesn't work with libstdc++-dev: you get the old 
```
/llvm/libdevice/imf_impl_utils.hpp:12:10: fatal error: 'cstddef' file not found
   12 | #include <cstddef>
```